plugins/generator-1.21.1/neoforge-1.21.1/templates/item/item.java.ftl

```diff
@@ -36,7 +36,7 @@
 package ${package}.item;
 
 <#compress>
-public class ${name}Item extends Item {
+<#if data.hasCustomJAVAModel() || data.getModels()?filter(e -> e.hasCustomJAVAModel())?has_content>@EventBusSubscriber(modid = "${modid}", bus = EventBusSubscriber.Bus.MOD)</#if> public class ${name}Item extends Item {

```

Comment:
This could be split into several lines to make the code more readable:
```suggestion
<#if data.hasCustomJAVAModel() || data.getModels()?filter(e -> e.hasCustomJAVAModel())?has_content>
@EventBusSubscriber(modid = "${modid}", bus = EventBusSubscriber.Bus.MOD)
</#if>
public class ${name}Item extends Item {
```